.github/workflows/build-and-test.yml

```
@@ -208,37 +150,73 @@ jobs:
           gem push pkg/*.gem
           rm -f $HOME/.gem/credentials
 
+  compute-tag:
+    runs-on: ubuntu-latest
+    outputs:
+      BRANCH_NAME: ${{ steps.branch_env.outputs.branch_name }}
+    steps:
+      - name: Set branch environment variables
+        id: branch_env
+        run: |
+          #BRANCH_NAME=${GITHUB_REF#refs/heads/}
```

Do we need to leave this in?
```suggestion
          #BRANCH_NAME=${GITHUB_REF#refs/heads/}
```